### PR TITLE
Fixed  rpc method bind error

### PR DESCRIPTION
**Pull request by MarvinFF**
fixed rpc_config return error mid
fixed  _rpc_bind _rpc_id_bind _rpc_unreliable_id_bind _rpc_unreliable_bind failed

### CODE REVIEW

File: scene/main/node.cpp [hunk 1]
```diff
@@ -555,10 +555,11 @@ Variant Node::_rpc_bind(const Variant **p_args, int p_argcount, Callable::CallEr
 		return Variant();
 	}
 
-	if (p_args[0]->get_type() != Variant::STRING) {
+	if (p_args[0]->get_type() != Variant::STRING_NAME) {
 		r_error.error = Callable::CallError::CALL_ERROR_INVALID_ARGUMENT;
 		r_error.argument = 0;
 		r_error.expected = Variant::STRING;
+
```
Comment (aaronfranke) [inline scene/main/node.cpp:562]: The surrounding methods don't have this extra line.

```suggestion
```

**Review by AndreaCatania — APPROVED** [review]
Looks good.

**Comment by Faless** [comment]
@MarvinFF can you please rebase and squash the 2 commits? See here for more info: https://docs.godotengine.org/en/stable/community/contributing/pr_workflow.html

**Comment by akien-mga** [comment]
I tried to force push a rebased squashed commit but did it wrong so it closed the PR.

So I pushed the commit manually to the `master` branch: 4c528e76e7.

Thanks for your contribution!

**Pull request closed without merging**